src/ui-test/server/ui/serversActivityBar.ts

```diff
@@ -0,0 +1,58 @@
+import { ActivityBar, SideBarView, ViewControl, TitleActionButton, VSBrowser } from 'vscode-extension-tester';
+import { AdaptersConstants } from '../../common/adaptersContants';
+import { RSPServerProvider } from './rspServerProvider';
+
+/**
+ * Servers activity bar representation
+ * @author Ondrej Dockal <odockal@redhat.com>
+ */
+export class ServersActivityBar {
+
+    private viewControl: ViewControl;
+    private sideBarView: SideBarView;
+
+    constructor() {
+        this.viewControl = new ActivityBar().getViewControl(AdaptersConstants.RSP_CONNECTOR_NAME);
+    }
+
+    async getSideBarView(): Promise<SideBarView> {
+        if (!this.sideBarView || !(await this.sideBarView.isDisplayed())) {
+            throw new Error("Servers side bar was not yet initializaed, call open first");
+        }
+        return this.sideBarView;
+    }
+    
+    public getViewControl(): ViewControl {
+        return this.viewControl;
+    }
+
+    async open(): Promise<SideBarView> {
+        if (!this.sideBarView || !(await this.sideBarView.isDisplayed())) {
+            this.sideBarView = await this.viewControl.openView();
```

Comment:
`viewControl.openView()` already checks whether the view is open

Reply:
true! done! thanks!